File: 110術科模擬試題/B1.java

import java.util.*;

public class B1 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		String input;// = scanner.nextLine();
		input = scanner.nextLine();
		int year=Integer.parseInt(input);
		if(year % 400 == 0)
			System.out.println("a leap year");
		else if( year % 4 ==0 && year % 100 !=0)
			System.out.println("a leap year");
		else
			System.out.println("a normal year");
	}
}

File: 110術科模擬試題/B1.vb
Imports System

Public Class Test
	Public Shared Sub Main()
		dim year as integer
		year=Integer.Parse(console.readline())
		if year mod 400 = 0 then
			console.writeline("a leap year")
		else if year mod 4 = 0 and year mod 100 <> 0 then
			console.writeline("a leap year")
		else
			console.writeline("a normal year")
		end if
	End Sub
End Class